x-pack/plugins/maps/public/shared/layers/sources/es_source.js

```diff
@@ -127,6 +141,21 @@ export class AbstractESSource extends AbstractVectorSource {
     }
   }
 
+  async supportsFitToBounds() {
+    try {
+      const geoField = await this._getGeoField();
+      if (geoField.type === 'geo_shape') {
```

Comment:
control-flow is a little heavy handed.

just do `return (geoField.type !== 'geo_shape')`.